### PR TITLE
Added-Category

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Product Category.
Screenshot (338)

### CODE REVIEW

File: src/Contexts/DataContext.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  useEffect(() => {
+    (async () => {
+      try {
+        const productResponse = await axios.get("/api/products");
+        setdata(productResponse.data.products);
+        const categoryResponse = await axios.get("/api/categories");
+        setCategoryData(categoryResponse.data.categories);
+      } catch (error) {
+        console.log(error);
+      }
```
Comment (JantuDeb) [inline src/Contexts/DataContext.js:19]: can destructure the `{data}`

**Review by JantuDeb — APPROVED** [review]
Overall the code looks fine

**Review by ShreyashMahajan — APPROVED** [review]
code looks fine to merge

File: src/Contexts/DataContext.js (hunk 1 above)
Comment (SaurabhChawda) [reply src/Contexts/DataContext.js:19]: Okay,I will Update.

**Comment by SaurabhChawda** [comment]
> code looks fine to merge

ThankYou

**Pull request merged**